### PR TITLE
Feature/dx-1141 List of banks for Sweden and Finland

### PR DESCRIPTION
**Pull request by arebra**
Added lists for Sweden and Finland

### CODE REVIEW

File: payments/trustly/index.md [hunk 1]
```diff
@@ -53,6 +53,47 @@ operations][after-payment] and the [payment resource][payment-resource].
...
+## Banks
+
+Trustly is supported by the following Finnish and Swedish banks as of August 14.
+2020.
+
+### Finland {% flag fi %}
+
+{:.table .table-striped}
+| :------------ | :------------------- | :-------------------- |
+| Bank   | Supported in eCom as of August 2020  |  Instant Credit Notification  |
```
Comment (asbjornu) [inline payments/trustly/index.md:65]:
```suggestion
| Bank   | Supported in eCom as of August 2020  |  Instant Credit Notification  |
| :------------ | :------------------- | :-------------------- |
```

File: payments/trustly/index.md [hunk 2]
```diff
@@ -53,6 +53,47 @@ operations][after-payment] and the [payment resource][payment-resource].
...
+| Ålandsbanken  | {% icon check %}   | {% icon check %}    |
+| S-Pankki  | {% icon check %}   | {% icon check %}    |
+| Danske Bank (Sampo)  | {% icon check %}   | {% icon check %}    |
+| Oma Säästöpankki  | {% icon check %}   | {% icon check %}    |
+
+### Sweden {% flag se %}
+
+{:.table .table-striped}
+| :------------ | :------------------- | :-------------------- |
+| Bank   | Supported in eCom as of August 2020  |  Instant Credit Notification  |
```
Comment (asbjornu) [inline payments/trustly/index.md:81]:
```suggestion
| Bank   | Supported in eCom as of August 2020  |  Instant Credit Notification  |
| :------------ | :------------------- | :-------------------- |
```

File: payments/trustly/other-features.md [hunk 3]
```diff
@@ -32,6 +32,8 @@ documentation_section="trustly" show_status_operations=true %}
 In Trustly Payments, you can create one type of payment and you can inspect and alter the details of the
 individual transactions within the payment.
 
+Hei
+
```
Comment (asbjornu) [inline payments/trustly/other-features.md:36]:
```suggestion
```

**Review by asbjornu — CHANGES_REQUESTED** [review]
Great! 😃 👍 

Do you think you can format the tables so columns are aligned? And perhaps the columns with the check marks should be centered?

File: payments/trustly/other-features.md (hunk 3 above)
Comment (arebra) [reply payments/trustly/other-features.md:36]: DOH!

File: payments/trustly/index.md (hunk 1 above)
Comment (arebra) [reply payments/trustly/index.md:65]: I'll try! I goofed something and can't get the localhost to start, so this check-in was blind.

**Comment by arebra** [comment]
Did it work?

**Review by marinafrydenb — APPROVED** [review]
format the tables, otherwise its nice :)

**Pull request merged**